template/src/components/navbar/navbar.html

```diff
@@ -15,7 +15,7 @@
     <collapse id="navbar" class="navbar-collapse" v-model="showNavbar">
       <ul class="nav navbar-nav">
         <li v-for="link in links" v-bind:class="{'active' : $route.path == link.path}">
-          <router-link v-bind:to="link.path">\{{link.name}}</router-link>
+          <router-link v-bind:to="link.path">{{link.name}}</router-link>
```

Comment:
Apologies but the escape character has to remain for the template to work. The `vue-cli` template engine also uses `{{}}` for variable replacement so the escape character needs to be ignored when a new project is initialised.

Reply:
I've used this boilerplate for production service last week, and I found some issues and bugs inside the boilerplate.
every change you see now is needed in order make this service work. :)

Reply:
It was tested on production-ready product, and it works as I used it.
notice that I've changed many things, such as the http-server, and some major configurations in order to make it work, also as a docker container.

Reply:
you're right.
it was fixed now.